dso-l2/src/main/java/com/tc/l2/state/ConsistencyMBean.java

```diff
@@ -43,4 +46,9 @@
    * @return the list of transitions requested.
    */
   Collection<Transition> requestedActions();
+
+  /**
+   * Forcibly allow a requested transition action
+   */
+  void forceTransition();
```

Comment:
This belongs in its own MBean

Reply:
Kept this here as this would provide a unified way to get a server out of suspended mode on startup. The old override vote mechanism won't work for a server suspended during startup
even in consistency mode. So I thought I'll add it here so that the signalling is uniform for both availability and consistency modes.